.gitignore

```diff
@@ -1 +1,2 @@
+__pycache__/
```

Comment:
I'd prefer not to add these kinds of changes, which are unrelated to the purpose of this pull request. For a longer discussion on this topic, see: https://github.com/pelican-plugins/image-process/pull/45#issuecomment-827423816

Reply:
Ok, reverting this